refined/shared/src/main/scala/monocle/refined/strings.scala

```diff
@@ -1,16 +1,16 @@
 package monocle.refined
 
-import eu.timepit.refined.api.Refined
+import eu.timepit.refined.api.{Validate, Refined}
 import eu.timepit.refined.string.StartsWith
 import monocle._
 
 object strings extends StringsInstances
 
 trait StringsInstances {
 
-  def startsWith(prefix: String): Prism[String, StartsWithString[prefix.type]] = {
+  def startsWith(prefix: String)(implicit v: Validate[String, StartsWith[prefix.type]]): Prism[String, StartsWithString[prefix.type]] = {
```

Comment:
just to be sure I understand correctly, we cannot fetch a `Validate[String, StartsWith[prefix.type]` inside the method definition because we need know `prefix`?